src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -305,7 +279,7 @@ public void BindingWorkflow_SilentSaveSolutionIfDirty()
         private BindingWorkflow CreateTestSubject(ProjectInformation projectInfo = null)
         {
             var useProjectInfo = projectInfo ?? new ProjectInformation { Key = "key" };
-
+            this.sonarQubeService.SetConnection(new Uri("http://connected"));
```

Comment:
Binding workflow now uses a connection that is referenced at the beginning of the flow (otherwise you may think - why do I do all this and check for the connection only now?).